tests/queries/0_stateless/03461_pk_prefix_trivial_count.reference

```diff
@@ -0,0 +1,7 @@
+-- { echo ON }
+
+DROP TABLE IF EXISTS t;
+CREATE TABLE t(k String) ORDER BY k as select 'dst_'||number from numbers(1e6);
```

Comment:
it's reproducible even with 10 rows `numbers(10)`
I guess 1e6 is too much for this test